src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/product.yml

```diff
@@ -66,6 +66,111 @@ admin_products_v2_delete:
   requirements:
     productId: \d+
 
+admin_products_v2_duplicate:
+  path: /{productId}/duplicate
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Product:duplicate
+    _legacy_controller: AdminProducts
+    _legacy_link:
+      - AdminProducts:duplicate
+    _legacy_parameters:
+      id_product: productId
+  requirements:
+    productId: \d+
+
+admin_products_v2_preview:
+  path: /{productId}/preview
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Product:preview
+    _legacy_controller: AdminProducts
+    _legacy_link:
+      - AdminProducts:preview
+    _legacy_parameters:
+      id_product: productId
+  requirements:
+    productId: \d+
+
+
+admin_products_v2_toggle_status:
+  path: /{productId}/toggle_status
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Product:toggleStatus
+    _legacy_controller: AdminProducts
+    _legacy_link:
+      - AdminProducts:statusproduct
+    _legacy_parameters:
+      id_product: productId
+  requirements:
+    productId: \d+
+
+admin_products_v2_update_position:
+  path: /update_position
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Product:updatePosition
+    _legacy_controller: AdminProducts
+    _legacy_link:
+      - AdminProducts:positionproduct
+    _legacy_parameters:
+      id_product: productId
+  requirements:
+    productId: \d+
+
+admin_products_v2_bulk_enable:
+  path: /bulk-enable
+  methods: [ POST, DELETE ]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Product:bulkEnable
+    _legacy_controller: AdminProducts
+    _legacy_link:
+      - AdminProducts:submitBulkdisableSelectionproduct
+    _legacy_parameters:
+      id_product: productId
+  requirements:
+    productId: \d+
+
+admin_products_v2_bulk_disable:
+  path: /bulk-disable
+  methods: [ POST, DELETE ]
```

Comment:
Did you get those permissions from a legacy controller, it feels a bit weird to use the DELETE request here, semantically even if the product is disabled it's still not deleted (for the enable route it makes even less sense)